### ISSUE TITLE
Questie error for the quest On Spirit's Wings

### ISSUE THREAD

**Issue by Thaowyr** [Questie/Questie#2914]
Hi, questie 6.3.14 was working fine until the quest On Spirit's Wings in Blade Edge is game me this inane error:

Questie: [ERROR] [QuestieQuest]: [Tooltips] There was an error populating objectives for Eavesdrop on the Bloodmaul ogres' plans 10714 0 No error
...erface\AddOns\Questie\Modules\Quest\QuestieQuest.lua:726: bad argument #1 to 'next' (table expected, got nil)
[string "@Interface\AddOns\Questie\Modules\Quest\QuestieQuest.lua"]:968: in function <...erface\AddOns\Questie\Modules\Quest\QuestieQuest.lua:966>
[string "=[C]"]: in function `next'
[string "@Interface\AddOns\Questie\Modules\Quest\QuestieQuest.lua"]:726: in function <...erface\AddOns\Questie\Modules\Quest\QuestieQuest.lua:683>
[string "=[C]"]: in function `xpcall'
[string "@Interface\AddOns\Questie\Modules\Quest\QuestieQuest.lua"]:966: in function <...erface\AddOns\Questie\Modules\Quest\QuestieQuest.lua:964>
[string "@Interface\AddOns\Questie\Modules\Quest\QuestieQuest.lua"]:1019: in function `PopulateObjectiveNotes'
[string "@Interface\AddOns\Questie\Modules\Quest\QuestieQuest.lua"]:509: in function `GetAllQuestIds'
[string "@Interface\AddOns\Questie\Modules\QuestieInit.lua"]:148: in function <Interface\AddOns\Questie\Modules\QuestieInit.lua:45>

Questie: [ERROR] [QuestieQuest]: v6.3.14 - There was an error populating objectives for On Spirit's Wings 10714 1 No error

**Comment by Thaowyr** [comment]
When I did the quest I got the following error:

Questie: [ERROR] [QuestieQuest]: [Tooltips] There was an error populating objectives for Eavesdrop on the Bloodmaul ogres' plans 10714 0 No erro
...erface\AddOns\Questie\Modules\Quest\QuestieQuest.lua:773: bad argument #1 to 'next' (table expected, got nil)
[string "@Interface\AddOns\Questie\Modules\Quest\QuestieQuest.lua"]:556: in function <...erface\AddOns\Questie\Modules\Quest\QuestieQuest.lua:554>
[string "=[C]"]: in function `next'
[string "@Interface\AddOns\Questie\Modules\Quest\QuestieQuest.lua"]:773: in function <...erface\AddOns\Questie\Modules\Quest\QuestieQuest.lua:772>
[string "@Interface\AddOns\Questie\Modules\Quest\QuestieQuest.lua"]:710: in function <...erface\AddOns\Questie\Modules\Quest\QuestieQuest.lua:683>
[string "=[C]"]: in function `xpcall'
[string "@Interface\AddOns\Questie\Modules\Quest\QuestieQuest.lua"]:554: in function `UpdateObjectiveNotes'
[string "@Interface\AddOns\Questie\Modules\Quest\QuestieQuest.lua"]:469: in function `UpdateQuest'
[string "@Interface\AddOns\Questie\Modules\Libs\QuestieHash.lua"]:108: in function <...nterface\AddOns\Questie\Modules\Libs\QuestieHash.lua:103>
[string "@Interface\AddOns\Questie\Modules\Libs\QuestieHash.lua"]:91: in function `CompareQuestHashes'
[string "@Interface\AddOns\Questie\Modules\QuestieEventHandler.lua"]:296: in function <...rface\AddOns\Questie\Modules\QuestieEventHandler.lua:278>
[string "=[C]"]: ?
[string "@Interface\AddOns\Quartz\libs\AceBucket-3.0\AceBucket-3.0.lua"]:73: in function <...e\AddOns\Quartz\libs\AceBucket-3.0\AceBucket-3.0.lua:71>
[string "@Interface\AddOns\Quartz\libs\AceBucket-3.0\AceBucket-3.0.lua"]:89: in function `func'
[string "@Interface\AddOns\Details\Libs\AceTimer-3.0\AceTimer-3.0.lua"]:57: in function <...ce\AddOns\Details\Libs\AceTimer-3.0\AceTimer-3.0.lua:50>

**Comment by dashihawk** [comment]
Here is the error I got


Questie: [ERROR] [QuestieQuest]: [Tooltips] There was an error populating objectives for Eavesdrop on the Bloodmaul ogres' plans 10714 0 No error 
...erface\AddOns\Questie\Modules\Quest\QuestieQuest.lua:773: bad argument #1 to 'next' (table expected, got nil) 
[string "@Interface\AddOns\Questie\Modules\Quest\QuestieQuest.lua"]:556: in function <...erface\AddOns\Questie\Modules\Quest\QuestieQuest.lua:554>
[string "=[C]"]: in function `next'
[string "@Interface\AddOns\Questie\Modules\Quest\QuestieQuest.lua"]:773: in function <...erface\AddOns\Questie\Modules\Quest\QuestieQuest.lua:772>
[string "@Interface\AddOns\Questie\Modules\Quest\QuestieQuest.lua"]:710: in function <...erface\AddOns\Questie\Modules\Quest\QuestieQuest.lua:683>
[string "=[C]"]: in function `xpcall'
[string "@Interface\AddOns\Questie\Modules\Quest\QuestieQuest.lua"]:554: in function `UpdateObjectiveNotes'
[string "@Interface\AddOns\Questie\Modules\Quest\QuestieQuest.lua"]:469: in function `UpdateQuest'
[string "@Interface\AddOns\Questie\Modules\Libs\QuestieHash.lua"]:108: in function <...nterface\AddOns\Questie\Modules\Libs\QuestieHash.lua:103>
[string "@Interface\AddOns\Questie\Modules\Libs\QuestieHash.lua"]:91: in function `CompareQuestHashes'
[string "@Interface\AddOns\Questie\Modules\QuestieEventHandler.lua"]:296: in function <...rface\AddOns\Questie\Modules\QuestieEventHandler.lua:278>
[string "=[C]"]: ?
[string "@Interface\AddOns\Questie\Libs\AceBucket-3.0\AceBucket-3.0.lua"]:73: in function <...\AddOns\Questie\Libs\AceBucket-3.0\AceBucket-3.0.lua:71>
[string "@Interface\AddOns\Questie\Libs\AceBucket-3.0\AceBucket-3.0.lua"]:89: in function `func'
[string "@Interface\AddOns\AuctionMaster\libs\Ace3\AceTimer-3.0\AceTimer-3.0.lua"]:57: in function <...uctionMaster\libs\Ace3\AceTimer-3.0\AceTimer-3.0.lua:50>

**Comment by Intervencion** [comment]
Came to post this as well :)

```
[02:26:16] Questie: [ERROR] [QuestieQuest]: [Tooltips] There was an error populating objectives for Eavesdrop on the Bloodmaul ogres' plans 10714 0 No error 
[02:26:16] ...erface\AddOns\Questie\Modules\Quest\QuestieQuest.lua:726: bad argument #1 to 'next' (table expected, got nil) 
[02:26:16] [string "@Interface\AddOns\Questie\Modules\Quest\QuestieQuest.lua"]:968: in function <...erface\AddOns\Questie\Modules\Quest\QuestieQuest.lua:966>
[string "=[C]"]: in function `next'
[string "@Interface\AddOns\Questie\Modules\Quest\QuestieQuest.lua"]:726: in function <...erface\AddOns\Questie\Modules\Quest\QuestieQuest.lua:683>
[string "=[C]"]: in function `xpcall'
[string "@Interface\AddOns\Questie\Modules\Quest\QuestieQuest.lua"]:966: in function <...erface\AddOns\Questie\Modules\Quest\QuestieQuest.lua:964>
[string "@Interface\AddOns\Questie\Modules\Quest\QuestieQuest.lua"]:1019: in function `PopulateObjectiveNotes'
[string "@Interface\AddOns\Questie\Modules\Quest\QuestieQuest.lua"]:379: in function `val'
[string "@Interface\AddOns\Questie\Modules\TaskQueue.lua"]:8: in function <Interface\AddOns\Questie\Modules\TaskQueue.lua:6>
 
[02:26:16] Questie: [ERROR] [QuestieQuest]: v6.3.14 - There was an error populating objectives for On Spirit's Wings 10714 1 No error 
```

**Comment by drejjmit** [comment]
Fixed in [v6.3.15](https://github.com/Questie/Questie/releases)
